chord_metadata_service/experiments/schemas.py

```diff
@@ -1,10 +1,15 @@
 from .descriptions import EXPERIMENT, EXPERIMENT_RESULT, INSTRUMENT
 from chord_metadata_service.restapi.schemas import ONTOLOGY_CLASS_LIST, KEY_VALUE_OBJECT
 from chord_metadata_service.restapi.schema_utils import tag_ids_and_describe
+from chord_metadata_service.ontologies import readXsdSimpleTypeValues
 
 
 __all__ = ["EXPERIMENT_SCHEMA", "EXPERIMENT_RESULT_SCHEMA", "INSTRUMENT_SCHEMA"]
 
+LIBRARY_STRATEGIES = readXsdSimpleTypeValues(
+    'chord_metadata_service/ontologies/xsd/SRA.experiment.xsd.xml',
```

Comment:
maybe use pathlib for constructing this path - that way its a bit more resilient

also, this file should be added to the pyproject `include` block (keep it alphabetical order)!

Reply:
also, we should put a note in README about the provenance of this XML file + where to find the license (in the header of the xml...)

Reply:
finally, a comment explaining what we're doing here would be good